src/issue/screens/issue.screen.js

```diff
@@ -230,7 +230,7 @@ class Issue extends Component {
       navigation,
     } = this.props;
 
-    const isLoadingData = isPendingComments || isPendingIssue;
+    const isLoadingData = isPendingComments || isPendingIssue || false;
```

Comment:
I was actually having problems on that line. For some reason, I was receiving `undefined` on both `isPendingComments` and `isPendingIssue` and the OR of two `undefined` is `undefined`. That would usually not be an issue because of JS's truthsy falsy, but the `refreshing` prop on `<FlatList />` requires a `boolean` specifically. 

Looking over the file again... I am not sure how either of those props can be `undefined`. They both come from `issueReducer` and they both have default values. 

As a solution to this, I can prepend the comparator with `!!` to force it to be a `boolean`, but that would just be avoiding this `undefined` problem I kinda want to get to the root of.